demisto_sdk/tests/test_files/content_slim/Packs/Sample02/Classifiers/classifier-sample2_packs_new.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
you can remove this file, classifier file will not go in both `packs` and `new`

Reply:
Removed